python/python/ert_gui/simulation/models/ensemble_smoother.py

```diff
@@ -30,7 +31,7 @@ def runSimulations(self, arguments):
         self._job_queue = self._queue_config.create_job_queue( )
         num_successful_realizations = self.ert().getEnkfSimulationRunner().runSimpleStep(self._job_queue, prior_context)
 
-        self.checkHaveSufficientRealizations(num_successful_realizations)
+        self.assertHaveSufficientRealizations(num_successful_realizations , active_realizations)
```

Comment:
Hmmm - must say I am a bit skeptical to this change. This check should be of whether we have enough (whatever that might be) realisations to perform the smoother update, and that question should be fully independent of how many realisations you started with.